code-challenges.js

```diff
@@ -8,54 +8,185 @@
+//350 is above boiling point
+//212 is at boiling point
+
+// I had to keep swtiching the conditional statements but I eventually got the ouput that I wanted. I originally had if and else if statements because I was following the example in the syllabus. I'm still trying to understand the differences between them and how and when to use them. I can break down the function in words but putting it together in code has been a struggle for me since jumpstart. 
+
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the variables provided below. Expected output: 10
 
 // Pseudo code:
 
+//problem: combining two arrays and return how many elements are in the new combined array.
+//function expression: use .length on new array to return the number of elements in the new array
+//set parameters: both arrays need to be added together in order to get the total number of elements.
+//make a decision: use the addition symbol to add the length of each array. Use ".length" after each variable to get the length of each array. 
+//return: the total number of elements of merged arrays console.log(myNumbers1.length + myNumbers2.length)
+
 const myNumbers1 = [3, 7, 0, 36, -9]
 const myNumbers2 = [8, -7, 42, 9, 13]
 
+console.log(myNumbers1.length + myNumbers2.length)
+
+//Output: 10
+
+//I honestly did not think that would work. I've told myself to stop over thinking and try the simplest way to work the problem first then start branching out.
+
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "2202 atleD"
 
 // Pseudo code:
 
+//problem: the letters in the string need to be reversed
+//function expression: I'm going to try .reverse() to see if that works with the string. If not then I'll try charsReversed.
+//set parameters: the letters in the string need to be be in reverse order
+//make decision: using .reverse() should work to return the letters in reverse order 
+//return: the letters in the string to be reversed. 
+    //After trying both of those and doing more research on w3 and the syllabus, I've decided to try indexing the string and then reversing it.
+    //That did not work so I will try joining and then reversing the string.
+    //Also did not work so I will do the opposite of join and split the string.
+
 const currentCohort = "Delta 2022"
+console.log(currentCohort.split(""))
+
+var cohortSplit = currentCohort.split("")
+    console.log(cohortSplit.reverse(), cohortSplit.join(""))
+return(cohortSplit.reverse(), cohortSplit.join(""))
+//Output: 2202 atleD
+
+//This took me about an hour and a half to figure it out. I figured out I had to split and join and reverse the values. I went back to the syllabus to see which order to put it in and also went back to the practice we did in class. It was pretty frustrating that the code didn't work like how i thought it was going to the first time.
```

Comment:
This can definitely be another one that can be easy to overthink.  Sometimes it is hard to come up with the correct combination, especially when built in methods are very finicky on what data type they work on like reverse().